### DISCUSSION TITLE
Location Issues in Student Developer Pack Application

### DISCUSSION THREAD

**Discussion by Lisagmrk** [community/community#66007] [GitHub Education]
### Hi everyone,

Hi,

I'm a student and I want to verify my student status in order to get access to the student developer pack. I've submitted countless applications already, but they keep getting rejected because "the school you selected does not appear to have a campus location in your country" and "you appear not to be near any campus location for the school you have selected". However, I have submitted the applications from different locations already, including from the exact location of my university, but I still keep getting the same error no matter where I am. I have provided my student email and my dated student ID. How can I fix this issue and get access?

**Comment by ssk-12** [comment]
Hey hi,

I am facing the same issue now, if you have found the fix, please share it here.
That would be helpful 🙂

**Comment by rohitkrsaini** [comment]
Hi 
I am facing the same problem
